src/main/java/pl/byczazagroda/trackexpensesappbackend/service/WalletServiceImpl.java

```diff
@@ -29,7 +32,19 @@ public WalletDTO updateWallet(UpdateWalletDTO dto) throws ResourceNotFoundExcept
 
         wallet.setName(dto.name());
 
-        return mapper.mapWalletEntityToWalletDTO(wallet);
+        return walletModelMapper.mapWalletEntityToWalletDTO(wallet);
     }
 
+    @Override
+    public WalletDTO createWallet(CreateWalletDTO createWalletDTO) {
+        String walletName = createWalletDTO.name();
+        Wallet wallet = new Wallet(walletName);
+        Wallet savedWallet = walletRepository.save(wallet);
+        boolean isWalletExists = walletRepository.existsById(savedWallet.getId());
+
+        if (isWalletExists) {
+            return walletModelMapper.mapWalletEntityToWalletDTO(savedWallet);
+        }
+        throw new ResourceNotSavedException("Sorry. Something went wrong and your Wallet is not saved. Contact administrator.");
```

Comment:
Please change it to: "Sorry. Something went wrong and your Wallet is not saved. **Please** contact administrator."

Reply:
Done.